spec/factories/divisions.rb

```diff
@@ -22,6 +22,9 @@
   factory :division do
     description { Faker::Lorem.sentence }
     name { Faker::Company.name }
+    internal_name nil
+    parent nil
```

Comment:
It felt useful to me to explicitly assign the nil, in particular for the 'parent' field, so it's clear that the field wasn't inadvertently left out.  I also noticed an explicit 'nil' in your basic_projects.rb factory, so I assumed that was an acceptable convention.